SQLFlowInterpreter.java

```diff
@@ -0,0 +1,94 @@
+package org.apache.zeppelin.sqlflow;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterException;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.sqlflow.client.MessageHandlerZeppelin;
+import org.sqlflow.client.SQLFlow;
+import org.apache.zeppelin.sqlflow.client.utils.EnvironmentSpecificSQLFlowClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.lang3.StringUtils;
+
+public class SQLFlowInterpreter extends Interpreter {
+	// Print Log
+	private static final Logger LOGGER = LoggerFactory
+			.getLogger(SQLFlowInterpreter.class);
+
+	private Map<String, String> parameters = new HashMap<String, String>();
+
+	public static final String SQLFLOW_SERVERADDR = "sqlflow.serverAddr";
+	public static final String MYSQL_USERNAME = "mysql.username";
```

Comment:
Do we only support mysql databases?

Reply:
We don't just support mysql database. For other SQL engines, including Oracle, Hive, SparkSQL, Flink, etc., developers can expand and improve according to actual needs.

Reply:
Shall we still use `mysql.username` to specific the hive username ?

Reply:
This initial version has not yet opened the connection to the hive database.

Reply:
> This initial version has not yet opened the connection to the hive database.

Let's do this under the new [issue](https://github.com/sql-machine-learning/zeppelin-sqlflow/issues/23).